File: January 8, 2026.py
def is_sorted(arr):

    asc = True
    desc = True

    for i in range(len(arr) - 1):
        if arr[i] > arr[i + 1]:
            asc = False
        elif arr[i] < arr[i + 1]:
            desc = False

    if asc:
        return "Ascending"
    elif desc:
        return "Descending"

    return "Not sorted"

File: January 8, 2026.js
function isSorted(arr) {

  let asc = true;
  let desc = true;

  for (let i = 0; i < arr.length - 1; i++) {
    if (arr[i] > arr[i + 1]) {
      asc = false;
    } else if (arr[i] < arr[i + 1]) {
      desc = false;
    }
  }

  if (asc) return "Ascending";
  if (desc) return "Descending";

  return "Not sorted";
}